### PR TITLE
Signifyd branch

### PR DESCRIPTION
**Pull request by tbricca**
By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Describe the purpose of this PR along with any background information and the impacts of the proposed change. For the benefit of the community, please do not assume prior context.
>NOTE: This has NOT been tested yet. I did not have an API key, so there may be issues when getting a response from the API.

> Provide details that support your chosen implementation, including: breaking changes, alternatives considered, changes to the API, etc.
>
> If the UI is being changed, please provide screenshots.

### References

> Include any links supporting this change such as a:
>
> - GitHub Issue/PR number addressed or fixed
> - Auth0 Community post
> - StackOverflow post
> - Support forum thread
> - Related pull requests/issues from other repos
>
> If there are no references, simply delete this section.

### Testing

> Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.
>
> Also include details of the environment this PR was developed in (language/platform/browser version).

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used, if not `master`

### CODE REVIEW

File: src/rules/signifyd-score.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/**
+ * @title Detect Ecommerce Fraud Users 
+ * @overview Get the signifyd score from signfyd.com and store it on app_metadata.
+ * @gallery true
+ * @category enrich profile
+ *
+ *
+ * This rule gets the signifyd score and status from signifyd.com and stored it in app_metadata.
```
Comment (seejee) [inline src/rules/signifyd-score.js:8]: Typo here: stored -> stores

File: src/rules/signifyd-score.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      //      "headline": Maxine Trycia
+      //      "orderId": 19418
+      //      "orderAmount": 48
+      //      "associatedTeam": 1
+      // ]
+      user.app_metadata.signifyd_details = signifyd.data.details;
+  
+      auth0.users.updateAppMetadata(user.user_id, user.app_metadata)
+        .then(function(){
+          context.idToken['https://example.com/signifyd_status'] = user.app_metadata.signifyd_status;
```
Comment (seejee) [inline src/rules/signifyd-score.js:54]: I don't see anywhere above where we set `user.app_metadata.signifyd_status`.

File: src/rules/signifyd-score.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    
+    }, function (err, resp, body) {
+      if (err) return callback(null, user, context);
+      if (resp.statusCode !== 200) return callback(null, user, context);
+      const signifyd = JSON.parse(body);
+      if (signifyd.status !== 'Ok') return callback(null, user, context);
+  
+      user.app_metadata = user.app_metadata || {};
+      user.app_metadata.guaranteeEligible = signifyd.data.status;
+      user.app_metadata.signifyd = signifyd.data.score;
```
Comment (seejee) [inline src/rules/signifyd-score.js:38]: Should this be `user.app_metadata.signifyd_score = signifyd.data.score` instead?

Comment (tbricca) [reply src/rules/signifyd-score.js:38]: Yes, good catch

**Pull request merged**